tmt/checks/avc.py

```diff
@@ -121,17 +126,18 @@ def _report_failure(label: str, exc: tmt.utils.RunError) -> list[str]:
             report += _report_failure('sestatus', exc)
 
         # Record selinux-policy NVR.
-        output, exc = _run_script(ShellScript('rpm -q selinux-policy'))
+        interesting_packages = ' '.join(INTERESTING_PACKAGES)
+        output, exc = _run_script(ShellScript(f'rpm -q {interesting_packages} || /bin/true'))
```

Comment:
with added `|| /bin/true` this script never fails? 

I would rather say both `selinux-policy` and `audit` are require for this check to work properly, so not sure why it should not fail

Reply:
Agreed, sounds reasonable. Otherwise the exception handling below would not make any sense.

Reply:
Fixed in 18d2e7a8